src/Transport/Administration/NamespacePermissions.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace NServiceBus.Transport.AzureServiceBus
+{
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.ServiceBus;
+    using Microsoft.Azure.ServiceBus.Management;
+    using Microsoft.Azure.ServiceBus.Primitives;
+
+    class NamespacePermissions
+    {
+        readonly ServiceBusConnectionStringBuilder connectionStringBuilder;
+        readonly ITokenProvider tokenProvider;
+
+        public NamespacePermissions(ServiceBusConnectionStringBuilder connectionStringBuilder, ITokenProvider tokenProvider)
+        {
+            this.connectionStringBuilder = connectionStringBuilder;
+            this.tokenProvider = tokenProvider;
+        }
+
+        public async Task<StartupCheckResult> CanManage()
+        {
+            var client = new ManagementClient(connectionStringBuilder, tokenProvider);
```

Comment:
Does the client need to be closed with `CloseAsync` as the finally block?

https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.servicebus.management.managementclient.closeasync?view=azure-dotnet

Reply:
Fixed in https://github.com/Particular/NServiceBus.Transport.AzureServiceBus/pull/29/commits/df4ee277e63a433f53370ceb7ae6a602141d988f